randovania/data/json_data/prime1.txt

```diff
@@ -2326,32 +2508,54 @@ Asset id: 445853540
 > Dock to Transport to Chozo Ruins South; Heals? False
   > Transport to Chozo Ruins South/Dock to Transport Tunnel D
       Trivial
+  > Transport Tunnel D/Dock to Great Tree Hall
+      Trivial
 
 > Dock to Great Tree Hall; Heals? False
   > Great Tree Hall/Dock to Transport Tunnel D
       Trivial
+  > Transport Tunnel D/Dock to Transport to Chozo Ruins South
+      Trivial
 
 ----------------
 Life Grove Tunnel
 Asset id: 3036397301
 > Dock to Great Tree Hall; Heals? False
   > Great Tree Hall/Dock to Life Grove Tunnel
       Trivial
+  > Life Grove Tunnel/Dock to Life Grove
+      Can Boost and Can lay bomb and Can lay power bomb
+  > Life Grove Tunnel/Pickup (Missile Expansion)
+      Can Boost and Can lay bomb and Can lay power bomb
 
 > Dock to Life Grove; Heals? False
   > Life Grove/Dock to Life Grove Tunnel
       Trivial
+  > Life Grove Tunnel/Dock to Great Tree Hall
+      Can Boost and Can lay bomb and Can lay power bomb
+  > Life Grove Tunnel/Pickup (Missile Expansion)
+      Can Boost
+
+> Pickup (Missile Expansion); Heals? False
+  > Life Grove Tunnel/Dock to Great Tree Hall
+      Can Boost and Can lay bomb and Can lay power bomb
+  > Life Grove Tunnel/Dock to Life Grove
+      Can Boost
 
 ----------------
 Transport Tunnel E
 Asset id: 2637367815
 > Dock to Transport to Phazon Mines East; Heals? False
   > Transport to Phazon Mines East/Dock to Transport Tunnel E
       Trivial
+  > Transport Tunnel E/Dock to Great Tree Hall
+      Trivial
 
 > Dock to Great Tree Hall; Heals? False
   > Great Tree Hall/Dock to Transport Tunnel E
       Trivial
+  > Transport Tunnel E/Dock to Transport to Phazon Mines East
+      Trivial
 
 ----------------
 Transport to Chozo Ruins South
```

Comment:
Add another node to this room called "Elevator to Chozo Ruins - Transport to Tallon Overworld South" with the requirement Scan Visor.